This is not applicable upstream as this breaks rebuilding translations whenever it changes, and we don't want that since we just building it once for packaging, not actively developing on it.

So keep this here until https://github.com/getting-things-gnome/gtg/issues/720 is solved

diff --git a/GTG/plugins/meson.build b/GTG/plugins/meson.build
index 7ecb530..f6b2744 100644
--- a/GTG/plugins/meson.build
+++ b/GTG/plugins/meson.build
@@ -23,9 +23,7 @@ foreach plugin : gtg_plugins
     install_dir: plugin_install_dir,
     type: 'desktop',
     args: ['--keyword=name', '--keyword=short-description', '--keyword=description'],
-    build_by_default: true,
-    build_always_stale: true
-    # build always because otherwise new translations won't be applied
+    build_by_default: true
   )
   subdir(plugin.underscorify())
 endforeach
